components/views/ScanConnect.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Stack, Text } from '@chakra-ui/react'
+import ViewLayout from '../ViewLayout'
+import QRCode from 'react-qr-code'
+import { Wallet } from '../../data/wallets'
+import { useConfig } from '../../contexts/ConfigContext'
+
+interface ScanConnectProps {
+  onBack: () => void
+  onCloseModal: () => void
+  wallet: Wallet
+}
+
+export default function ScanConnect({
+  onBack,
+  onCloseModal,
+  wallet,
+}: ScanConnectProps) {
+  const { walletConnectUri } = useConfig()
+  return (
+    <ViewLayout
+      header={{
+        title: `Connect to ${wallet.name}`,
```

Comment:
Same comment as last pr about casing if it comes through different than expected